assets/src/application/ui/input/InlineEditInput/TabbableText/index.tsx

```diff
@@ -17,7 +17,7 @@ const TabbableText: React.FC<TabbableTextProps> = ({ onRequestEdit, icon, text,
         }
     };
 
-    const tooltip = props.tooltip || __('Click to edit title...')
```

Comment:
it must be me making this silly mistake :facepalm:

Reply:
we all make silly mistakes from time to time